stages/connection-generator/src/main/java/edu/kit/kastel/mcse/ardoco/core/connectiongenerator/informants/ProjectNameInformant.java

```diff
@@ -51,6 +52,7 @@ public void run() {
             var recommendationState = recommendationStates.getRecommendationState(metamodel);
 
             checkForProjectNameInRecommendedInstances(projectName, recommendationState);
+            checkForProjectNameInRecommendedInstances(projectName.toLowerCase(), recommendationState);
```

Comment:
projectName is derived from ProjectPipelineData, which is populated from the ArDoCoRunner, which provides the name in its constructor. However, these classes are sparsely documented. For the tests, the name is either in some cases hardcoded as a constant `"teastore"` in the test class or derived from the enum name. Since the enum name is always UPPERCASE, it is immediately converted to lowercase in most classes before being passed to the runner. 

This is very easy to miss. If you do not provide the name in lowercase, the project name informant will not filter the project name correctly, leading to a large worsening in the metrics. I can attest to this, since at one point i missed converting it to lowercase before feeding it to the runner and spend the next 60+ minutes trying to figure out why my metrics are terrible.

However, i think it is more sensible to adjust the ProjectNameInformant so that it ignores case when searching for similar words. I will do that and remove line 55, if that's fine